Rakefile

```diff
@@ -17,4 +17,4 @@ Refinery::Testing::Railtie.load_dummy_tasks(ENGINE_PATH)
 
 load File.expand_path('../tasks/rspec.rake', __FILE__)
 
-task :default => :spec
+task default: :spec
```

Comment:
preferably we don't change existing code unless we're already changing that line.